### PR TITLE
✨Adding/porting some basic commands 

### PR DESCRIPTION
**Pull request by Killeroo**
_Bunny online_

### CODE REVIEW

**Comment by eltone** [comment]
Where does this code run? Is it a long running process or start on demand? These stats will reset when the process stops right?

**Comment by eltone** [comment]
It feels like I'm haunting you in your free time @Killeroo.

**Comment by Killeroo** [comment]
> 
> 
> Where does this code run? Is it a long running process or start on demand? These stats will reset when the process stops right?

Long running process, yeah they will reset when the process stops. Not perfect but haven't got a super simple lazy bones method for persistent stats accross multiple processes/builds at the moment. This will do for now 🥇

File: LeafBot/LeafBot/Commands/Games.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+using LeafBot.Data;
+
+namespace LeafBot.Commands
+{
+  public class Games : BaseCommandModule
+  {
+    private Random random = new Random();
+
+    [Command("roll")]
+    [Description("Instructs LeafBot to roll a lettice shaped dice")]
```
Comment (izzylys) [inline LeafBot/LeafBot/Commands/Games.cs]: how dare you make a spelling error in leaf bot >:(

File: LeafBot/LeafBot/Program.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ namespace LeafBot
 {
   class Program
   {
-    public const string VERSION = "1.0";
```
Comment (izzylys) [inline LeafBot/LeafBot/Program.cs]: wow, you were very bold

**Review by izzylys — APPROVED** [review]
other than the fact that you spelled "lettuce" as "lettice" this looks good to me! 

APPROVED!

File: LeafBot/LeafBot/Commands/Games.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      Dictionary<int, string> roles = new Dictionary<int, string>
+      {
+        {1, $"Tank {tankEmoji}"},
+        {2, $"Damage {damageEmoji}"},
+        {3, $"Healer {healerEmoji}" }
+      };
+
+      // Choose random number of roles to pick, randomly order roles, pick random number of items and order based on key
+      int numberOfRoles = random.Next(1, 4);
+      var picks = roles.OrderBy(x => random.Next()).Take(numberOfRoles).OrderBy(x => x.Key).ToArray();
```
Comment (izzylys) [inline LeafBot/LeafBot/Commands/Games.cs:68]: ah wait soz premature approve. to confirm, you're wanting to randomly order roles right? if so, the second order by is reordering your randomly ordered roles. it should just be:

```csharp
var picks = roles.OrderBy(x => random.Next()).Take(numberOfRoles).ToArray();
```

**Review by izzylys — CHANGES_REQUESTED** [review]

File: LeafBot/LeafBot/Commands/Games.cs (hunk 3 above)
Comment (Killeroo) [reply LeafBot/LeafBot/Commands/Games.cs:68]: Interesting, so I wanted to randomly order the list, pick x roles then re-order the resulting list based on their key. This was so I could get the roles in the same rough order as they appear in overwatch, a bit ott but this solution seemed to work.

Comment (izzylys) [reply LeafBot/LeafBot/Commands/Games.cs:68]: ah ok my bad then, I misunderstood your intention! agreed it's a bit ott, but if it works as you intended then it's good enough for government work

**Review by izzylys — APPROVED** [review]

**Pull request merged**